### ISSUE TITLE
Clicking while opening causes 'cancel'

### ISSUE THREAD

**Issue by johnpaulmedina** [t4t5/sweetalert#605]
```
            closeOnCancel: false,
            closeOnConfirm: false,
            allowOutsideClick: false,
```

While these options are set, clicking while opening a sweet alert causes the cancel action.

**Comment by kriddile** [comment]
Removing the "|| targetedOverlay" from [this line](https://github.com/t4t5/sweetalert/blob/master/dist/sweetalert-dev.js#L422) seems to fix this problem.  Doing so likely breaks allowOutsideClick, but I always have that option set to false anyways.

**Comment by t4t5** [comment]
This should be fixed in SweetAlert 2.0, with the new [closeOnClickOutside option](https://sweetalert.js.org/docs/#closeonclickoutside)
